lib/chromedriver.js

```diff
@@ -226,12 +225,22 @@ class Chromedriver extends events.EventEmitter {
     } else {
       cmd = `pkill -15 -f "${this.chromedriver}.*--port=${this.proxyPort}"`;
     }
-    log.info(`Killing any old chromedrivers, running: ${cmd}`);
+    log.debug(`Killing any old chromedrivers, running: ${cmd}`);
     try {
-      await (B.promisify(cp.exec))(cmd);
-      log.info("Successfully cleaned up old chromedrivers");
+      await B.promisify(cp.exec(cmd));
+      log.debug("Successfully cleaned up old chromedrivers");
     } catch (err) {
-      log.info("No old chromedrivers seemed to exist");
+      log.warn("No old chromedrivers seemed to exist");
+    }
+    log.debug(`Clean any old adb forwarded port socket connections`);
+    let connections = await this.adb.getForwardList();
+    for (let conn of connections) {
+      if (conn.indexOf('webview_devtools') !== -1) {
```

Comment:
can we add a comment about why "webview_devtools" string is a magic constant here?

Reply:
Yes, It's better for understanding